web-common/src/metrics/service/ErrorEventFactory.ts

```diff
@@ -26,6 +27,14 @@ export interface ErrorEvent extends MetricsEvent {
   glob: boolean;
 }
 
+export interface ErrorBoundaryEvent extends MetricsEvent {
+  action: ErrorEventAction;
+  screen_name: MetricsEventScreenName;
+  api: string;
+  status: string;
+  message: string;
+}
```

Comment:
Think it'd make sense to split this into something like the following two types?
1. `HTTPErrorEvent` – to be used in our TanStack Query error handler, where the `api` field is relevant.
2. `JavascriptErrorEvent` – to be used in the Javascript error listeners, where a `stacktrace` field would be relevant.